assets/i18n/locales/nl.json

```diff
@@ -0,0 +1,54 @@
+{
+  "button/cancel": "Annuleren",
+  "button/ok": "Bevestigen",
+  "button/push_locations": "Push",
+  "connectivity/action_requires_connection": "De gevraagde actie vereist een actieve netwerkverbinding",
+  "export/body": "Hier zijn mijn gegevens van Sharebibles",
+  "export/error_title": "Exporteren mislukt",
+  "export/generic_error": "Er is een onverwachte fout opgetreden. Probeer het opnieuw.",
+  "export/no_email_error": "U heeft geen e-mailtoepassing! Installeer er een zodat u uw gegevens kunt exporteren.",
+  "export/subject": "Mijn Sharebibles gegevens",
+  "feedback/feedback_error": "Kan e-mailapp niet starten. Verzend feedback handmatig naar {{email}}",
+  "feedback/feedback_subject": "Share Bibles Feedback",
+  "feedback/feedback_title": "Feedback versturen",
+  "invites/accept": "Accepteren",
+  "invites/error_title": "Uitnodigingsprobleem",
+  "invites/error": "Er is een probleem opgetreden bij het accepteren van uw toegangstoken. Probeer het opnieuw en/of neem contact op met {{email}}",
+  "invites/placeholder": "Uitnodiging",
+  "permissions/camera/requesting": "Kunnen we alsjeblieft je camera gebruiken om uitnodigingen te accepteren?",
+  "permissions/camera/denied": "Schakel de camera in de instellingen in om uitnodigingen via QR-code toe te staan.",
+  "pin_callout/bibles_heading": "Bijbels",
+  "pin_callout/last_visited": "Laatst bezocht {{visited_time}}",
+  "pin_callout/status_heading": "Status",
+  "resource/english_bible_esv": "# bijbels gedistribueerd",
+  "resource/generic_bible": "# bijbels gedistribueerd",
+  "settings/accept_invites": "Uitnodiging accepteren",
+  "settings/change_language": "Taal:",
+  "settings/export": "Data exporteren",
+  "settings/invite": "Uitnodiging delen",
+  "settings/logout": "Uitloggen",
+  "settings/not_implemented": "Sorry deze functie is nog niet beschikbaar.",
+  "settings/push_locations_message": "Door locatiegegevens te pushen, worden alle lokaal opgeslagen locatiegegevens naar de centrale database gestuurd zodat anderen deze kunnen openen. Wilt u uw gegevens pushen?",
+  "settings/push_locations": "locatiegegevens pushen",
+  "settings/region": "Regio: {{region}}",
+  "settings/send_feedback": "Stuur feedback",
+  "settings/shareInvite": "Deel een Share Bibles toegangstoken",
+  "settings/token_success": "Bedankt voor het invoeren van uw toegangstoken!",
+  "settings/token_title": "Toegangstoken",
+  "status/accepted": "Aangenomen",
+  "status/delivered": "Afgeleverd",
+  "status/groups/home": "Thuis",
+  "status/groups/not_home": "Niet Thuis",
+  "status/have": "Heeft bijbel",
+  "status/need": "Anders",
+  "status/not_home": "Niet thuis",
+  "status/rejected": "geweigerd",
+  "status/unfinished": "Opnieuw",
+  "status/unknown": "Onbekend",
+  "title/accept_invites": "Uitnodiging accepteren",
+  "title/share_bibles": "Sharebibles",
```

Comment:
@Robwiebe I switched this back to the standard title. If you want to change it back to your previous term we can do it in another PR.